ext/MitsubaLoader/CMitsubaLoader.cpp

```diff
@@ -378,6 +378,23 @@ CMitsubaLoader::SContext::group_ass_type CMitsubaLoader::loadShapeGroup(SContext
 	return mesh;
 }
 
+//TODO : vtx input and assembly params are now ignored (mb is created without pipeline), later they need to be somehow forwarded and set on already created pipeline
+static core::smart_refctd_ptr<ICPUMesh> createMeshFromGeomCreatorReturnType(IGeometryCreator::return_type&& _data)
```

Comment:
`createMeshStub` its a stub cause it unusable without a pipeline

Maybe take in the pipeline as well as the `_data` ?